docs/tutorials/11_quadratic_hamiltonian_and_slater_determinants.ipynb

```diff
@@ -432,7 +497,7 @@
    "name": "python",
    "nbconvert_exporter": "python",
    "pygments_lexer": "ipython3",
-   "version": "3.9.6"
+   "version": "3.6.8"
```

Comment:
I actually do run them with a newer kernel, and only switch it back before committing because my kernel is named which breaks the CI. It's the same issue as before...